### PR TITLE
Add onelogin instructions and reformat authenticate doc

### PR DESCRIPTION
**Pull request by gongjason**
* Add instructions for Onelogin
* reformat main authenticate instructions

### CODE REVIEW

**Comment by gongjason** [comment]
@jamilbk flagging so you have a chance to take a look before you create the next release. If no major changes, it'd be good to get this in there :)

File: docs/docs/authenticate/index.md [hunk 1]
```diff
@@ -54,34 +69,32 @@ default['firezone']['authentication']['oidc'] = {
...
 identity providers:
 
-* [Azure AD]({%link docs/authenticate/azure-ad.md%})
-* [Google]({%link docs/authenticate/google.md%})
-* [Okta]({%link docs/authenticate/okta.md%})
-* [Local email/password authentication (default)]({%link docs/authenticate/web-auth.md%})
+<!-- markdownlint-disable MD013 -->
+<!-- markdownlint-disable MD034 -->
+
+| OIDC provider | offline_access | Link to Guide                                        |
```
Comment (jamilbk) [inline docs/docs/authenticate/index.md]: Kinda confused about this table -- is it a feature matrix? Should the `offline_access` header be renamed to like `User Synchronization` or something, and then the fields updated to be like

> 'offline_access' required in `scope` parameter

**Review by jamilbk — CHANGES_REQUESTED** [review]
@gongjason Nvmd will followup over chat

**Comment by jamilbk** [comment]
Needs a bit further investigation to see if the `refresh_token` will be provided somehow.

**Pull request closed without merging**